libnavui-maneuver/src/main/java/com/mapbox/navigation/ui/maneuver/view/MapboxManeuverView.kt

```diff
@@ -745,4 +752,11 @@ class MapboxManeuverView : ConstraintLayout {
     internal fun getUpcomingManeuverAdapter(): MapboxUpcomingManeuverAdapter {
         return upcomingManeuverAdapter
     }
+
+    override fun onSizeChanged(w: Int, h: Int, oldw: Int, oldh: Int) {
+        super.onSizeChanged(w, h, oldw, oldh)
+        if (h != oldh) {
+            _heightFlow.tryEmit(h)
+        }
```

Comment:
Comparing with the old value is not necessary, because `MutableStateFlow` doesn't notify collectors, if the value hasn't changed. 
```suggestion
        _heightFlow.value = h
```

Reply:
Not applicable anymore.